woocommerce/woocommerce-android
# Code scanner UI in the order listing screen

Closes: #9055 


### Description

This PR adds an entry point to the "Scanned and Deliver" project in the Order listing screen. It basically adds a barcode icon in the toolbar in the order listing screen.

More context: pecCkj-GT-p2

<img width="452" alt="Screenshot 2023-05-17 at 1 03 00 PM" src="https://github.com/woocommerce/woocommerce-android/assets/1331230/22eeb613-0af0-4f03-9d5a-88f6b9c1aa75">


### Testing instructions

1. Navigate to the orders listing screen
2. Ensure you see the barcode icon in the toolbar
3. Tap on the barcode icon
4. Ensure the scanning screen opens


### Images/gif

<img width="300" src="https://github.com/woocommerce/woocommerce-android/assets/1331230/e842ecd4-2922-4802-9177-e467f7cd98bd"/>

<img width="300" src="https://github.com/woocommerce/woocommerce-android/assets/1331230/7a27c73a-5b82-4245-b970-89124e6dad1a"/>
